app/controllers/album.js

```diff
@@ -0,0 +1,14 @@
+const fetch = require('node-fetch');
+
+const listAlbums = (req, res) => {
```

Comment:
Missing `next` callback, that parameter is used for pass to next middleware, in controller we use that function for passing errors, for example:
```
service.function().then(() => ).catch(err => next(err))
```

Reply:
Return promise in controller function